sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/GenericResourceExpanded.cs

```diff
@@ -9,14 +9,21 @@ namespace Azure.ResourceManager.Core
     /// <summary>
     /// A class representing an expanded generic azure resource along with the instance operations that can be performed on it.
     /// </summary>
-    public class GenericResourceExpanded : GenericResourceExpandedOperations
+    public class GenericResourceExpanded : GenericResourceOperations
```

Comment:
Can we duplicate the test cases in GenericResourceOperationsTests to get back an GenericResourceExpanded and try each of the methods to makes sure they still work, should be able to use all the exact same asserts, just the beginning would be different since we need to start with an expanded